### PR TITLE
Convert ViewHolder to Kotlin

### PR DESCRIPTION
**Pull request by anticafe**
#### Issue tracker   
- Convert all ViewHolder to Kotlin

#### Release  
Addresses release/3.6
  
#### Test cases
- [ ] Covered
  
#### Manual testing
- [x] Done  
  
If yes,  
- Device: Galaxy S10
- OS: Android 11

#### Build tasks success  
Successfully running following tasks on local 
- [x] `./gradlew assembledebug`
- [x] `./gradlew spotlessCheck`

#### Additional Info

### CODE REVIEW

File: app/src/main/java/com/amaze/filemanager/adapters/holders/CompressedItemViewHolder.kt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.amaze.filemanager.R
+import com.amaze.filemanager.ui.views.RoundedImageView
+import com.amaze.filemanager.ui.views.ThemedTextView
+
+/** @author Emmanuel Messulam<emmanuelbendavid></emmanuelbendavid>@gmail.com> on 17/9/2017, at 18:13.
```
[the remark this answers is not in the document]
Comment (anticafe) [reply app/src/main/java/com/amaze/filemanager/adapters/holders/CompressedItemViewHolder.kt]: Updated in f329c318a9bcac00d3f6133e72dde39ba50521fc

File: app/src/main/java/com/amaze/filemanager/adapters/holders/AppHolder.kt [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import android.view.View
+import android.widget.ImageButton
+import android.widget.ImageView
+import android.widget.RelativeLayout
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.amaze.filemanager.R
+import com.amaze.filemanager.ui.views.ThemedTextView
+
+/** @author Emmanuel Messulam <emmanuelbendavid></emmanuelbendavid>@gmail.com> on 10/12/2017, at 14:45. */
```
[the remark this answers is not in the document]
Comment (anticafe) [reply app/src/main/java/com/amaze/filemanager/adapters/holders/AppHolder.kt]: Updated in f329c318a9bcac00d3f6133e72dde39ba50521fc

**Review by VishalNehra — APPROVED** [review]

**Comment by anticafe** [comment]
@VishalNehra Do you know why the **Android Feature CI / build (pull_request)** job failed with error 500?

**Comment by VishalNehra** [comment]
https://github.com/actions/checkout/issues/505
Just retrigger the build

**Review by TranceLove — APPROVED** [review]
I'm OK with it. +1 and merge.

**Pull request merged**